src/plot/curve.rs

```diff
@@ -1,4 +1,5 @@
 //! `kurobako plot curve` command.
+#![allow(clippy::format_push_string)]
```

Comment:
Clippy says "`String += format!()` shoud be replaced with `write!()`". However, this replacement isn't trivial and there is no performance problem relating to this point in the current code. So, I decided to just ignore this check.